src/traces/scattergl/convert.js

```diff
@@ -192,10 +192,10 @@ function convertMarkerStyle(trace) {
         // See  https://github.com/plotly/plotly.js/pull/1781#discussion_r121820798
         if(multiLineWidth) {
             for(i = 0; i < count; i++) {
-                borderSizes[i] = markerSizeFunc(optsIn.line.width[i]);
+                borderSizes[i] = optsIn.line.width[i] / 2;
             }
         } else {
-            s = markerSizeFunc(optsIn.line.width);
+            s = optsIn.line.width / 2;
```

Comment:
Before this commit, `gl2d_transforms` looked like:

![image](https://user-images.githubusercontent.com/6675409/40630791-98757ffa-62a3-11e8-90de-a67855c6eb95.png)